src/platform/forms/save-in-progress/SaveInProgressIntro.jsx

```diff
@@ -142,6 +143,11 @@ class SaveInProgressIntro extends React.Component {
     } else if (renderSignInMessage) {
       alert = renderSignInMessage(prefillEnabled);
     } else if (prefillEnabled && !verifyRequiredPrefill) {
+      // If you pass in a headingLevel prop it will change what is rendered by the changeSaveInProgressHeadingLevel helper
+      const saveInProgressHeader = changeSaveInProgressHeadingLevel(
+        this.props.headingLevel,
```

Comment:
@bkjohnson Good catch! I added it to `propTypes` but I changed it so that the default in the switch statement is an `h3` so no `defaultProps` is required.

Reply:
Can you tell me why the switch statement is necessary? I don't feel like it is providing a lot of value for the additional complexity it introduces compared to the other two options I presented.

Reply:
Additionally, the request for a default prop was as much for documentation and easier understanding as it was for functional reasons. Someone who opens this file may see `headingLevel` as a prop, but have no idea what/if there is a default value for it. By ensuring that there is a default and it is located in the same file along with all the other defaults, a developer can immediately know what it is without having to guess or go hunting in another file.

Reply:
@bkjohnson Yep, I added the default prop into my refactor